docs/superpowers/plans/2026-06-01-tier3-sidecar-consolidation.md

```
@@ -0,0 +1,718 @@
+Add the dispatch arm in `run()`:
+
+```rust
+        Some("build-embedded-base") => {
+            let args = parse_build_embedded_base_args(argv)?;
+            run_build_embedded_base(args).await
+        }
+```
+
+Update the `None =>` usage string to include `build-embedded-base`, and add a `build-embedded-base` line to `print_help`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)